api/api/spec/spec.yaml

```diff
@@ -8262,6 +8276,40 @@ paths:
         '429':
           $ref: '#/components/responses/TooManyRequestsResponse'
 
+  /agents/uninstall:
+    get:
+      tags:
+        - Agents
+      summary: "Check user's permission to uninstall agents"
+      description: "Verify if the user has the necessary permissions to uninstall agents."
+      operationId: api.controllers.agent_controller.get_agent_uninstall_permission
+      x-rbac-actions:
+        - $ref: '#/x-rbac-catalog/actions/agent:uninstall'
+      parameters:
+        - $ref: '#/components/parameters/pretty'
+        - $ref: '#/components/parameters/wait_for_complete'
+      responses:
+        '200':
+          description: "User has the necessary permissions to uninstall agents"
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/ApiResponse'
+              example:
+                message: "User has permission to be uninstalled."
```

Comment:
```suggestion
                message: "User has permission to uninstall agents."
```

Reply:
Changed in b7dcc227e48732b77db2c92c1087e322a55d8ae9.